src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

```diff
@@ -69,16 +69,78 @@ All the `http_req_...` lines and the ones after them are _built-in_ metrics that
+| ----------- | ---- | ----------- |
+| vus                | Gauge   | Current number of active virtual users |
+| vus_max            | Gauge   | Max possible number of virtual users (VU resources are pre-allocated, to ensure performance will not be affected when scaling up the load level) |
+| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function). |
+| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well. See the workaround in the example below, how to create a sub-metric for getting only the duration of the iteration's function for a specific scenario. |
 | dropped_iterations | Counter | Introduced in k6 v0.27.0, the number of iterations that could not be started due to lack of VUs (for the arrival-rate executors) or lack of time (due to expired maxDuration in the iteration-based executors). |
-| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                 |
-| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                     |
-| checks             | Rate    | The rate of successful checks.                                                                                                                                                                                  |
+| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| checks             | Rate    | The rate of successful checks. |
+
+<Collapsible title="Workaround to calculate `iteration_duration` metric only for a scenario">
+
+A common requested case is to track the `iteration_duration` metric without including time spent for `setup` and `teardown` functions.
+This feature is not yet available but a threshold on `iteration_duration` or any pre-existing metrics can be used as a workaround.
+
+It's based on the concept of creating sub-metrics for the required scope and set always-pass thresholds. It mostly works with every filter that already works with Threshold, for example:
+* 'iteration_duration{scenario:default}' will generate a sub-metric collecting samples only for the default scenario's iteration. 'scenario:default' because that's the internal k6 scenario name when we haven't actually specified `options.scenarios` explicitly and are just using the execution shortcuts instead.
```

Comment:
```suggestion
* 'iteration_duration{scenario:default}' generates a sub-metric collecting samples only for the default scenario's iteration. 'scenario:default' is used because that's the internal k6 scenario name when we haven't specified `options.scenarios` explicitly and are just using the execution shortcuts instead.
```